livestream-compose-sample/src/main/java/io/getstream/livestream/compose/Utils.kt

```diff
@@ -21,3 +26,63 @@ fun Context.randomDescription(): Int {
         descArray.recycle()
     }
 }
+
+private const val SECOND_MILLIS = 1000
+private const val MINUTE_MILLIS = 60 * SECOND_MILLIS
+private const val HOUR_MILLIS = 60 * MINUTE_MILLIS
+private const val DAY_MILLIS = 24 * HOUR_MILLIS
+
+fun Context.getTimeAgo(date: Date): String {
+    val now: Date = Calendar.getInstance().time
+    val diff: Long = now.time - date.time
+    return when {
+        diff < SECOND_MILLIS -> {
+            getString(R.string.just_now)
+        }
+        diff < MINUTE_MILLIS -> {
+            String.format(getString(R.string.seconds_ago), diff / SECOND_MILLIS)
+        }
+        diff < 2 * MINUTE_MILLIS -> {
+            String.format(getString(R.string.minutes_ago), 1)
+        }
+        diff < 59 * MINUTE_MILLIS -> {
+            String.format(getString(R.string.minutes_ago), diff / MINUTE_MILLIS)
+        }
+        diff < 90 * MINUTE_MILLIS -> {
+            String.format(getString(R.string.hour_ago), diff / HOUR_MILLIS)
+        }
+        diff < 48 * HOUR_MILLIS -> {
+            getString(R.string.yesterday)
+        }
+        diff < 6 * DAY_MILLIS -> {
+            String.format(getString(R.string.hour_ago), diff / DAY_MILLIS)
+        }
+        diff <8 * DAY_MILLIS -> {
+            getString(R.string.week_ago)
+        }
+        else -> {
+            getString(R.string.long_time)
+        }
+    }
+}
+
+@Composable
+fun Context.cardBackground(): Color {
+    return if (isDarkTheme())
```

Comment:
> Make sure to use Kotlin best practices and semantics instead of the java-based approach.
Noted, updated the signature of the function 

>  this either shouldn't be a @Composable function if we're using it as an extension to a Context, or it shouldn't be an extension to a Context and it should use LocalContext.current value to fetch the data.

Updated it to be simple `@Composable` function with local context , no longer an extension function